deb-package-builder/build.sh

```diff
@@ -65,11 +68,25 @@ for FULL_VERSION in $(echo ${PHP_VERSIONS} | tr "," "\n"); do
             gcp-php-${PHP_VERSION}-${PHP_VERSION}
         popd
     fi
-    sed -i -e "s/PHP_VERSION/${PHP_VERSION}/" debian/control debian/rules
-    sed -i -e "s/SHORT_VERSION/${SHORT_VERSION}/" debian/control debian/rules \
-        debian/patches/series
+    envsubst '${PHP_VERSION},${SHORT_VERSION}' < debian/rules > debian/rules.tmp
```

Comment:
Consider naming the original (template) file something like `debian/rules.in`. Same with the control and patches files below.